Author: Michael R. Crusoe <crusoe@debian.org>
Description: Skip myst-parser & pydata-sphinx-theme
Forwarded: not-needed

They are not yet in Debian
--- ipyparallel.orig/docs/source/conf.py
+++ ipyparallel/docs/source/conf.py
@@ -44,7 +44,7 @@
     'sphinx.ext.inheritance_diagram',
     'sphinx.ext.intersphinx',
     'sphinx.ext.napoleon',
-    'myst_parser',
+#   'myst_parser',
     'nbsphinx',
     'IPython.sphinxext.ipython_console_highlighting',
 ]
@@ -153,7 +153,7 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'pydata_sphinx_theme'
+#html_theme = 'pydata_sphinx_theme'
 
 
 # Theme options are theme-specific and customize the look and feel of a theme
